Makefile

```
@@ -168,6 +172,10 @@ $(O)/compiler_spec: $(DEPS) $(SOURCES) $(SPEC_SOURCES) LLVM_CONFIG
 	@mkdir -p $(O)
 	$(EXPORT_CC) $(EXPORTS) ./bin/crystal build $(FLAGS) $(SPEC_WARNINGS_OFF) -o $@ spec/compiler_spec.cr
 
+$(O)/primitives_spec: $(O)/crystal $(DEPS) $(SOURCES) $(SPEC_SOURCES) LLVM_CONFIG
+	@mkdir -p $(O)
```

After #11509 this needs to be changed:
```suggestion
$(O)/primitives_spec: $(O)/crystal $(DEPS) $(SOURCES) $(SPEC_SOURCES)
        $(call check_llvm_config)
	@mkdir -p $(O)
```

---

To be fair, this one can be dropped entirely because `$(O)/crystal` will pick it up transitively

---

If `.build/crystal` already exists and is fresh, its recipe wouldn't execute.
But I suppose `primitive_spec` itself shouldn't need `LLVM_VERSION` so yeah, we're be good without it.